### PR TITLE
Drush switch msa

### PR DESCRIPTION
**Pull request by nlisgo**

### CODE REVIEW

File: src/modules/jcms_migrate/migration_assets/subjects.json [hunk 1]
```diff
@@ -97,6 +97,13 @@
     "image": "subjects/elife-sciences-biophysics-and-structural-biology-illustration.jpg",
     "alt": "Biophysics and Structural Biology"
   },
+  {
+    "id": "biochemistry-chemical-biology",
```
Comment (nlisgo) [inline src/modules/jcms_migrate/migration_assets/subjects.json]: I would prefer if we use migrate to create the subject in production. At least until I refactor how we ensure they are there on a clean install.

Comment (giorgiosironi) [reply src/modules/jcms_migrate/migration_assets/subjects.json]: Preferring that to creating it manually from the UI? I second this opinion.

**Comment by nlisgo** [comment]
Is there any reason that we can't create the new MSA on production now? That should be the only thing holding this up. We need this on `continuumtest--journal-cms` to verify the switch workflow.

File: src/modules/jcms_migrate/migration_assets/subjects.json (hunk 1 above)
Comment (nlisgo) [reply src/modules/jcms_migrate/migration_assets/subjects.json]: > Preferring that to creating it manually from the UI?

Yes

**Comment by giorgiosironi** [comment]
No idea, would it show in some list as an empty subject? @sianroderick?

Maybe https://github.com/elifesciences/builder-private/pull/136 holds this up as Journal will hide it?

**Comment by thewilkybarkid** [comment]
Yep on both counts (deploying config change now).

File: src/modules/jcms_migrate/migration_assets/subjects.json [hunk 2]
```diff
@@ -97,6 +97,13 @@
...
+    "name": "Biochemistry and Chemical Biology",
+    "impact_statement": "eLife publishes research spanning chemical biology and enzymology to metabolism and membrane proteins. Decisions are made by <a href=\"https://elifesciences.org/about/people/biochemistry\">journal editors</a> who are active researchers in biochemistry.",
+    "image": "subjects/elife-sciences-biochemistry-illustration.jpg",
+    "alt": "Biochemistry and Chemical Biology"
+  },
```
Comment (giorgiosironi) [inline src/modules/jcms_migrate/migration_assets/subjects.json]: shouldn't this show 2 subjects being created?

Comment (nlisgo) [reply src/modules/jcms_migrate/migration_assets/subjects.json]: > shouldn't this show 2 subjects being created?

I'm not sure what you mean, sorry.

Comment (nlisgo) [reply src/modules/jcms_migrate/migration_assets/subjects.json]: Have just added `Immunology and Inflammation`

Comment (giorgiosironi) [reply src/modules/jcms_migrate/migration_assets/subjects.json]: That was it :+1:

**Pull request merged**